src/engine/test/source/hlp/hlp_test.cpp

```diff
@@ -119,3 +119,15 @@ TEST(hlpTests, url_parsing)
     ASSERT_EQ("https", result["url.scheme"]);
     ASSERT_EQ("user", result["url.username"]);
 }
+
+TEST(json_test, dummy_test)
```

Comment:
should update the test to have a better name.
We also need to decide on a scheme on the tests group names (we now have 'hlpTests', 'json_test', 'map_test')